.github/CODEOWNERS

```diff
@@ -0,0 +1,37 @@
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# @global-owner1 and @global-owner2 will be requested for
+# review when someone opens a pull request.
+*      @FreeRTOS/pr-bar-raiser
```

Comment:
This needs to be @aws/freertos-pr-bar-raisers

Reply:
updated.